src/ApplicationDefaultCredentials.php

```diff
@@ -353,4 +357,21 @@ private static function onGce(
 
         return (new GCECache($gceCacheConfig, $cache))->onGce($httpHandler);
     }
+
+    /**
+     * A function that returns the default logger if the GOOGLE_SDK_DEBUG_LOGGING
+     * environment variable is set. Returns null if not.
+     *
+     * @return null|LoggerInterface
+     */
+    public static function getDefaultLogger(): null|LoggerInterface
+    {
+        $loggingFlag = (string)getenv(self::SDK_DEBUG_FLAG);
```

Comment:
nit
```suggestion
        $loggingFlag = (string) getenv(self::SDK_DEBUG_FLAG);
```

I've added a CS rule for this in https://github.com/GoogleCloudPlatform/php-tools/pull/149